src/SprykerEco/Glue/UnzerRestApi/Processor/Mapper/CheckoutDataResponseMapper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi\Processor\Mapper;
+
+use Generated\Shared\Transfer\RestCheckoutDataResponseAttributesTransfer;
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+
+class CheckoutDataResponseMapper implements RestCheckoutDataResponseAttributesMapperInterface
```

Comment:
```suggestion
class RestCheckoutDataResponseAttributesMapper implements RestCheckoutDataResponseAttributesMapperInterface
```

Reply:
please do not forget to adjust respective factory method

Reply:
Fixed.